platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/HederaState.java

```diff
@@ -28,6 +28,11 @@
  * detail, and are happy with just the API provided by this interface.
  */
 public interface HederaState {
+    /**
+     * The name of the thread that handles transactions. For the sake of the app, to allow logging.
+     */
+    String TRANSACTION_HANDLING_THREAD_NAME = "<scheduler TransactionHandler>";
```

Comment:
That there is a single thread for transaction handling is implementation detail that should not be made public, let alone part of Hedera state definition (which has nothing to do with that platform feature).
Moreover, the constant itself is a hack to overcome poor design. The actual thread name is formed in `SequentialThreadTaskScheduler()` and depends on configuration, i.e. whether there is a single thread with that name depends on `transactionHandler` configuration in `PlatformSchedulersConfig`.

Reply:
This dependency on the literal was introduced in https://github.com/hashgraph/hedera-services/pull/8633

The `TransactionStateLogger` will need to be modified to not depend on the constant, if that's what we want to do. @jasperpotts pinging you, since that was originally your implementation

Reply:
I moved the constant to `StateLogger`, made it private and removed its usage elsewhere.